stdlib/public/core/String.swift

```diff
@@ -655,7 +655,7 @@ extension String {
   internal func _uppercaseASCII(_ x: UInt8) -> UInt8 {
     /// A "table" for which ASCII characters need to be upper cased.
     /// To determine which bit corresponds to which ASCII character, subtract 1
-    /// from the ASCII value of that character and divide by 2. The bit is set iff
+    /// from the ASCII value of that character and divide by 2. The bit is set if
```

Comment:
It’s not a typo - “iff” means “if and only if”.

Reply:
Got it, thanks